docs/resources/machine.md

```diff
@@ -53,6 +54,15 @@ Optional:
 
 - `create` (String)
 
+
+<a id="nestedatt--network_interfaces"></a>
+### Nested Schema for `network_interfaces`
+
+Read-Only:
+
+- `id` (Number)
```

Comment:
It is considered to be a good practice not to return database internal identifiers. I cannot think about use-case for ID, maybe it is better not to return it at all? Maybe in future we'll move to GUID for all the resources in MAAS